modules/sandboxed-containers-building-blocks.adoc

```diff
@@ -4,4 +4,8 @@
 
 [id="sandboxed-containers-building-blocks_{context}"]
 
-= Sandboxed containers building blocks
+= {sandboxed-containers-first} building blocks
+
+The {sandboxed-containers-operator} encapsulates all of the pieces from Kata containers. The {sandboxed-containers-operator} manages installation, lifecycle, and configuration tasks.
+
+The {sandboxed-containers-operator} uses the link:https://quay.io/isolatedcontainers/sandboxed-containers-operator[sandboxed-containers-operator] container image. The image contains the controller part of the Operator that monitors and manages `KataConfig`.
```

Comment:
The `KataConfig` resource? Object? Custom resource? 

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-object-references